src/globals/scss/_css--helpers.scss

```diff
@@ -13,6 +13,14 @@
     visibility: visible;
     white-space: nowrap;
   }
+
+  .bx--body {
```

Comment:
`.bx--body` class gets applied when `$css--helpers: true`, which is the same styles as `body` from `_css--body.scss`